http/legacy/influxqld_handler.go

```diff
@@ -142,11 +142,19 @@ func (h *InfluxqlHandler) handleInfluxqldQuery(w http.ResponseWriter, r *http.Re
 		}
 	}
 
+	formatString := r.Header.Get("Accept")
+	encodingFormat := influxql.EncodingFormatFromMimeType(formatString)
+	if encodingFormat == influxql.EncodingFormatUnknown {
+		h.Logger.Warn("request included unknown format in Accept header, using application/json", zap.String("format", formatString))
```

Comment:
Not sure there is value in logging invalid `Accept` headers. It is a client issue and could result in spamming the logs.

Reply:
:+1: I'll remove it